app/controllers/tufts/qr_status_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Tufts
+  class QrStatusController < ApplicationController
+    protect_from_forgery with: :null_session
+
+    def set_status
+      model = ActiveFedora::Base.find(params[:id])
+      model.qr_status = ['Batch Reviewed']
+      model.save
+      render json: { batch_reviewed: true }
+    end
+
+    def status
+      model = ActiveFedora::Base.find(params[:id])
+      status = if model.qr_status == ['Batch Reviewed']
```

Comment:
I think it would be a good idea to make this a method on the models, as well. We already have the reviewed string hard coded twice, so this is feeling likely to be fragile. Something like this:


```ruby
BATCH_REVIEWED_STRING = 'Batch Reviewed'.freeze

def batch_reviewed?
  model.qr_status == [BATCH_REVIEWED_STRING]
end

def mark_batch_reviewed
  model.qr_status = [BATCH_REVIEWED_STRING]
end
```